hack/setup_env.sh

```diff
@@ -40,11 +40,13 @@ function spectral() {
 }
 
 function assisted_service() {
```

Comment:
Who is using it? I see it only on `init` makefile target but I don't see anyone using that target.

Reply:
OpenShift CI

Reply:
The "Dockerfile.assisted-service-build"